### ISSUE TITLE
Valid semantic version is reported as not valid

### ISSUE THREAD

**Issue by paulomorgado** [mhutch/MonoDevelop.MSBuildEditor#154]
This definition:

```xml
  <PropertyGroup>
    <Version>0.1.0-preview03</Version>
  </PropertyGroup>
```

is reported as not being a valid version format:

![image](https://github.com/mhutch/MonoDevelop.MSBuildEditor/assets/470455/941c0a0c-6c5d-4333-88b2-1a70285d3923)

but it's a valid semantic version format.

**Comment by DamianEdwards** [comment]
This seems to have been fixed in latest CI build.

**Comment by KirillOsenkov** [comment]
Weird, I can reproduce in the latest main branch.

**Comment by Thieum** [comment]
The issue is in the latest version available: 2.9.443.2356

![image](https://github.com/mhutch/MonoDevelop.MSBuildEditor/assets/944894/59c570d9-11bb-4bd6-ac29-2c8d41ab9d28)
![image](https://github.com/mhutch/MonoDevelop.MSBuildEditor/assets/944894/ff5c4284-74c6-4091-bf48-29694836e7a3)
